Source/ResearchPal/ResearchTree.cs

```diff
@@ -359,6 +359,7 @@ public static void FixPositions()
                 {
                     //Log.Message("moving overlapping root node " + overlap.ToString() + " to " + new IntVec2(overlap.Pos.x, overlap.Pos.z + maxWidth).ToString() + " because it overlaps node " + root.ToString());
                     bestPos += Math.Max(maxWidth, 1); // this is important - maxWidth can easily be 0, breaking the whole thing
+                    rootYOffset++;
```

Comment:
Note, that it is possible that adding the same Math.Max(maxWidth, 1) instead of one may be better, but I have no issues with current version.